lang/chinese.lang

```diff
@@ -1194,6 +1194,19 @@ Subject: [subject]
  
 -- 
 This e-mail has been sent via forum on [forum_address]."""
+email_notify_inactive_user_subject = '<!-- TODO -->Inactivity notification [name]'
```

Comment:
Because of the fact, that this is a snipptet for use in e-mails and not in a HTML context, the "<!-- TODO -->" would get shown in the subject of the e-mail (also valid for all other languages with exception of English and German).  
On the other hand I've no clue, what to do instead to give notice to translators. A code comment in the line before might be a solution.

Reply:
Yes, your are right. I just took a look to the Chinese file. Here, several subjects contains the TODO. I have removed these comments and added a new comment above.